### PR TITLE
[libressl] update to 3.4.2

### PR DESCRIPTION
**Pull request by Neumann-A**

### CODE REVIEW

File: ports/libressl/portfile.cmake [hunk 1]
```diff
@@ -4,20 +4,20 @@ if(EXISTS "${CURRENT_INSTALLED_DIR}/include/openssl/ssl.h")
...
-set(LIBRESSL_HASH 11defdde8169d3653c24e149e698ffc5a8ead5ac0808111d1986cb11ef72e9912c463d4891d4635877021e73a8a045dbdbe5e83ec785a59150f170d2ca2031de)
+set(LIBRESSL_VERSION 3.4.2)
+set(LIBRESSL_HASH ae91a840b29330681dc2a4f55a9bd760e6fe1bdfb3399017aae3a16bd21b413e97cbb8ba504400f0a1f42757f6128b3fa763d06bae4fc9f2b9dbeea867a57ad2)
 
 vcpkg_download_distfile(
     LIBRESSL_SOURCE_ARCHIVE
-    URLS https://ftp.openbsd.org/pub/OpenBSD/LibreSSL/${PORT}-${LIBRESSL_VERSION}.tar.gz https://ftp.fau.de/openbsd/LibreSSL/${PORT}-${LIBRESSL_VERSION}.tar.gz
-    FILENAME ${PORT}-${LIBRESSL_VERSION}.tar.gz
-    SHA512 ${LIBRESSL_HASH}
+    URLS "https://ftp.openbsd.org/pub/OpenBSD/LibreSSL/${PORT}-${LIBRESSL_VERSION}.tar.gz https://ftp.fau.de/openbsd/LibreSSL/${PORT}-${LIBRESSL_VERSION}.tar.gz"
```
Comment (dg0yt) [inline ports/libressl/portfile.cmake:12]: 2 URLs in one quoted argument - Is this intentional?

**Review by dg0yt — COMMENTED** [review]
Perhaps CI should at least download, perhaps even build, what is supported but must not be installed (`skip`)?

File: ports/libressl/portfile.cmake (hunk 1 above)
Comment (Neumann-A) [reply ports/libressl/portfile.cmake]: good catch. I changed the quotes after I tested the local build.

**Comment by Neumann-A** [comment]
@dg0yt CI of a6bc04b says it is green

**Pull request merged**